packages/@o3r/testing/schematics/ng-add/index.ts

```diff
@@ -1,11 +1,21 @@
-import type { Rule } from '@angular-devkit/schematics';
+import type { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
 
 /**
  * Add Otter testing to an Angular Project
  *
  * @param options
  */
 export function ngAdd(): Rule {
-  /* ng add rules */
-  return () => {};
+  return async (_tree: Tree, context: SchematicContext) => {
+    try {
+      const { addVsCodeRecommendations } = await import('@o3r/schematics');
+      return addVsCodeRecommendations(['Orta.vscode-jest']);
+
+    } catch (e) {
+      context.logger.error(`[ERROR]: Adding @o3r/testing has failed.
+      If the error is related to missing @o3r dependencies you need to install '@o3r/core' to be able to use the mobile package. Please run 'ng add @o3r/core' .
```

Comment:
`o3r/testing` has no peer dep `o3r/core`. Here we can directly display the message that `o3r/schematics` is missing and to run an ng add on it.

Reply:
I am not sure to understand you message.
It's true that `o3r/testing` does not depend of `o3r/core` but the message remains correct. If `@o3r/schematics` is not found it is because it is not part of the dependencies and the issue will be fixed by executing `ng add @o3r/core`.
Do you see something I don't ?

Reply:
I think what Mircea is saying is that in that case we know that it fails because we're only missing `@o3r/schematics`.
We could recommend to `ng add` this package specifically and not `@o3r/core`  (or both) because it may happen that someone is not interested in getting every packages

Reply:
Add both in the message